lib/wallet/migrations.js

```diff
@@ -162,6 +162,90 @@ class MigrateChangeAddress extends AbstractMigration {
   }
 }
 
+/**
+ * Migrate account for new lookahead entry.
+ * Applies to WalletDB v0
```

Comment:
we are already on WalletDB v1, this should migrate us to v2:

https://github.com/handshake-org/hsd/blob/309df94ad5b2874944a32de63b449ac0a542068d/lib/wallet/walletdb.js#L65